### DISCUSSION TITLE
Scroll through ONLY windows on i3 scratchpad

### DISCUSSION THREAD

**Discussion by marinjurjevic** [davatorium/rofi#1516] [Q&A]
Hi,

i3 wm has a special workspace called [scratchpad](https://i3wm.org/docs/userguide.html#_scratchpad). 
Currently when running `-show window` modi in rofi, windows in scratchpad don't have assigned workspace number. 
![rofi_hidden_workspace](https://user-images.githubusercontent.com/17838694/141614394-0cc843e3-391e-4efe-88c1-2b5ca9a4c8d2.png)

I would like to be able scroll only through scratchpad windows. When I'm working I have a lot of windows in scratchpad which I would like to access quickly. Currently they are mixed with foreground windows and finding the window I need can take a moment.
So in case in the image above, only two URxvt windows would appear in window modi.

**Comment by DaveDavenport** [comment]
This is very specific to one particular window manager. It is also not 'supported' from the EWMH interface.

I suggest making a script to do this.

**Reply by marinjurjevic** [reply]
> I suggest making a script to do this.

How would I go around filtering these windows? Can I just filter windows without assigned workspace?
